From dace477c92ea3a9111cef73159a6a1fcaaf0f497 Mon Sep 17 00:00:00 2001
From: Matthias Clasen <mclasen@redhat.com>
Date: Mon, 15 Oct 2012 19:22:50 -0400
Subject: [PATCH] Add Win32 versions of some new content type APIs

This commit adds minimal versions of
g_content_type_get_symbolic_icon and
g_content_type_get_generic_icon_name.
https://bugzilla.gnome.org/show_bug.cgi?id=683642
---
 gio/gcontenttype-win32.c | 12 ++++++++++++
 1 file changed, 12 insertions(+)

diff --git a/gio/gcontenttype-win32.c b/gio/gcontenttype-win32.c
index 40c8cf4..e7e9785 100644
--- a/gio/gcontenttype-win32.c
+++ b/gio/gcontenttype-win32.c
@@ -244,6 +244,18 @@ g_content_type_get_icon (const gchar *type)
   return G_ICON (themed_icon);
 }
 
+GIcon *
+g_content_type_get_symbolic_icon (const gchar *type)
+{
+  return g_content_type_get_icon (type);
+}
+
+gchar *
+g_content_type_get_generic_icon_name (const gchar *type)
+{
+  return NULL;
+}
+
 gboolean
 g_content_type_can_be_executable (const gchar *type)
 {
-- 
1.8.0

